app/app/api/api_v1/endpoints/users.py

```diff
@@ -170,21 +170,28 @@ def update_user_profile_picture(
     db: Session = Depends(deps.get_db),
     user_id: int,
     image: UploadFile = File(...),
-    current_admin: models.Admin = Depends(deps.get_current_active_admin_with_permission("user")),
+    current_user: models.User = Depends(deps.get_current_active_user),
 ) -> Any:
     """
     Update a user's profile picture
     """
+    if current_user.id == user_id or (
+        (admin := crud.admin.get(db, current_user.id))
+        and schemas.AdminPermissions(admin.permissions).is_allowed("user")
+    ):
+        if image.content_type not in ("image/png", "image/jpeg"):
+            raise HTTPException(status_code=415, detail="Profile Pictures can only be png or jpg images")
```

Comment:
Small p in pictures, and PNG / JPG capital? (Just a thought, whatever you prefer)